application/models/neo_tag.py

```diff
@@ -0,0 +1,23 @@
+from .neo_ogm_model import NeoOGMModel
+from py2neo.ogm import RelatedFrom, Property
+
+
+class NeoTag(NeoOGMModel):
```

Comment:
NeoTags are the objects that keep the tag information. It has an extra method find_or_create_by as NeoTags should be unique in the database. I'll add a uniqueness validation aswell. So not using this method will throw exceptions if a Tag already exists.